delegator/prisma/schema.prisma

```diff
@@ -8,18 +8,59 @@ datasource db {
 }
 
 model Transaction {
```

Comment:
I think we should include the relation from `Transaction` to `Function` table. What do you think?